guidelines/MONAI-Workloads.md

```diff
@@ -0,0 +1,435 @@
+
+### Definitions {#definitions}
+
+* **Clinical Use Case:** Defines the high level clinical use case where AI could be helpful.  _Example: Emergency brain hemorrhage diagnosis_
+* **Clinical Workflow:** Defines the clinical workflow including the systems, data types and response times that describe the next level of detail for each clinical use case. 
+* **Computational Workload:** Compute workload types for managing and processing Clinical Workloads.  
+    * [Asynchronous Computational Workload](#asynchronous-computational-workload)
+    * [Asynchronous Result Computational Workload](#asynchronous-result-computational-workload)
+    * [Batch Computational Workload](#batch-computational-workload)
+    * [Streaming Computational Workload](#streaming-computational-workload)
+    * [Synchronous Computational Workload](#synchronous-computational-workload)
+
+## Clinical Use Cases {#clinical-use-cases}
+
+The clinical use cases introduced in this section are intended to be a non-exhaustive set of examples to provide enough clarity to help define scenarios where a MONAI Deploy application is used and a way to map those scenarios to workloads and components that developers use to understand compute needs and end to end performance constraints.  Each figure in this section is a sequence diagram described by the use case steps in the same section.
+
+The clinical workflow is defined above. _Example: Incoming Images sent from the instrument via DICOM Router or PACS get sent to be diagnosed.  AI algorithms help with Segmentation & Classification algorithms and create a new DICOM object including the results, so they can be uploaded to PACS.  Incoming study size: 500MB, response time requirement from imaging to result being available for radiologists 20 minutes.
+
+### Quality Assurance {#quality-assurance}
+
+#### Metadata Quality Assurance {#metadata-quality-assurance}
+
+While studies are routed through the hospital DICOM routing infrastructure, images are validated with classification algorithms. Validating the usability of an image or data. Does the metadata associated with an image match the image? For instance, is this a left or right femur? The label states one direction but the image suggests another. These workloads allow for image corrections or create implied routing paths for further AI processing.
+
+**Clinical Workflow**
+
+1. Studies get routed in a large hospital DICOM routing system, at peak hour, the hospital is seeing about 300 studies per hour.
+2. As series arrive to DICOM router, DICOM router sends selected slices out of the series an AI processing system
+3. The AI processing system selects the correct slice or two for classification based on DICOM metadata.
```

Comment:
My first question is on 43rd raw, "3. The AI processing system selects the correct slice or two for classification based on DICOM metadata."

Reply:
The AI processing system selects slices for classification based on DICOM metadata